src/app/core/public/api/impl/Component.ts

```diff
@@ -67,6 +67,38 @@ export abstract class ComponentImpl<
         );
     }
 
+    public setNumPorts(group: string, amt: number): boolean {
+        // TODO(leon) revisit this and decide on a functionality
```

Comment:
Like the other TODOs, it may be helpful to elaborate when this should be done, but since this one is specifically directed towards you then it is your call